react/src/components/SettingList.tsx

```diff
@@ -1,118 +1,96 @@
+      return false;
+    }
     return item.value !== item.defaultValue;
   };
 
-  const itemMaxWidth = 600;
-  const isLeftTab = !mobile;
-  const leftTabWidth = 200;
-
   return (
     <Flex
       direction="column"
       gap={'md'}
       align="stretch"
       style={{
-        flex: 1,
         padding: token.paddingContentHorizontal,
-        paddingLeft: isLeftTab ? 0 : token.paddingContentHorizontal,
-        paddingRight: token.paddingContentHorizontal,
-        maxWidth: itemMaxWidth + leftTabWidth + token.marginLG,
+        maxWidth: token.screenLG,
       }}
     >
-      <Flex
-        direction="row"
-        justify="start"
-        wrap="wrap"
-        gap={'xs'}
-        style={{
-          paddingLeft: isLeftTab ? token.paddingContentHorizontal : 0,
-        }}
-      >
-        {/* <Typography.Title
-          level={4}
-          style={{
-            margin: 0,
-            marginLeft: isLeftTab ? token.marginMD : 0,
-            padding: 0,
-            width: isLeftTab ? leftTabWidth - token.marginXS : undefined,
-          }}
-        >
-          {t('settings.Config')}
-        </Typography.Title> */}
+      <Flex justify="start" gap={'xs'}>
         <Input
           prefix={<SearchOutlined />}
-          placeholder={placeholder || t('settings.SearchPlaceholder')}
+          placeholder={t('settings.SearchPlaceholder')}
           onChange={(e) => setSearchValue(e.target.value)}
           value={searchValue}
-          style={{ flex: 1, minWidth: 50 }}
         />
-        <Checkbox onChange={(e) => setChangedOptionFilter(e.target.checked)}>
+        <Checkbox
+          onChange={(e) => setChangedOptionFilter(e.target.checked)}
+          style={{ whiteSpace: 'nowrap' }}
+        >
           {t('settings.ShowOnlyChanged')}
         </Checkbox>
-
-        {/* <Button
-            icon={<RedoOutlined />}
-            onClick={() => {
-              _.flatMap(settingGroup, (item) => item.settingItems).forEach(
-                (option) => {
-                  option?.setValue && option.setValue(option.defaultValue);
-                },
-              );
-            }}
-          >
-            {t('button.Reset')}
-          </Button> */}
+        <Button
+          icon={<RedoOutlined />}
+          onClick={() => {
+            _.flatMap(settingGroup, (item) => item.settingItems).forEach(
+              (option) => {
+                option?.setValue && option.setValue(option.defaultValue);
+              },
+            );
+          }}
+        >
+          {t('button.Reset')}
+        </Button>
       </Flex>
       <Tabs
-        tabPosition={isLeftTab ? 'left' : 'top'}
-        tabBarStyle={{
-          width: isLeftTab ? leftTabWidth : undefined,
-        }}
-        items={appendedSettingGroup.map((settingGroup) => {
+        tabPosition={tabDirection ? 'left' : 'top'}
+        tabBarStyle={{ minWidth: 200 }}
+        items={_.map(appendedAllSettingGroup, (settingGroup) => {
```

Comment:
How about always adding `filterItems.length`? 
Please use secondary text for count.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/9ab44b95-675b-4de9-81fd-3afd8dbfc47b.png)